Tools/buildstream/elements/sdk/libspiel.bst

```diff
@@ -0,0 +1,26 @@
+kind: meson
+
+build-depends:
+- freedesktop-sdk.bst:public-stacks/buildsystem-meson.bst
+- sdk/gi-docgen.bst
```

Comment:
I would expect this is not needed given you are passing `-Ddocs=false` below.

Reply:
It's needed for the libspeechprovider project.

Reply:
Then add `-Dlibspeechprovider:docs=false` to the list of Meson flags, maybe `-Dlibspeechprovider:introspection=false` as well.

It could be argued that we should have a `sdk/libspeechprovider.bst` and that we set the build options for it there, but I'm fine having it as subproject given only `libspiel` will use it.

Reply:
Yeah I tried already the separate recipe approach and that didn't work, it's a meson issue in libspiel i think.

Reply:
Ah, that's a bit of a bummer, but it's what we have to work with. Let's then merge this as-is. Thanks for clarifying and trying out the options.